nextflow.config

```diff
@@ -93,7 +85,7 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
-        channels               = ['conda-forge', 'bioconda', 'defaults']
+        conda.channels         = ['conda-forge', 'bioconda']
```

Comment:
thanks for fixing this!

Reply:
Particularly keen not to have the `defaults` channel.